examples/streaming/streamingkmeans/src/main/resources/application.conf

```diff
@@ -0,0 +1,6 @@
+gearpump {
+  serializers {
+    "io.gearpump.streaming.examples.streamingkmeans.InputMessage" = ""
+    "io.gearpump.streaming.examples.streamingkmeans.ResultMessage" = ""
+  }
+}
```

Comment:
Try to change this config file name to geardefault.conf, then I think user does not need to put the conf file to folder ```gearpump/conf``` any more.